create_project.yml

```diff
@@ -17,13 +17,14 @@
 
     - name: Make sure the project exists in OpenShift and is up-to-date
       k8s:
-        api_version: v1
-        kind: Project
-        name: "{{ project_name }}"
-        resource_definition:
-          description: "{{ project_description|default(project_display_name)|default(project_name) }}"
-          display_name: "{{ project_display_name|default(project_name) }}"
```

Comment:
There seem to be a change in the format as well that is not described in the commit message?

Reply:
`definition` is an alias for `resource_definition`. I preferred moving all fields for this object as `definition` field children instead of mixing `definition` and `k8s` Ansible module arguments. This change is stated in the commit message by:

> changed the object definition to make it more explicit.